sdk/communication/Azure.Communication.CallAutomation/src/Generated/CallConnectionRestClient.cs

```diff
@@ -598,7 +598,7 @@ public async Task<Response<MuteParticipantResult>> MuteAsync(string callConnecti
             await _pipeline.SendAsync(message, cancellationToken).ConfigureAwait(false);
             switch (message.Response.Status)
             {
-                case 200:
+                case 202:
```

Comment:
Were these from someone else's changes? Did we want these included?

Reply:
Yeah - this was from a separate team and it's autogenerated. The tests were modified accordingly to pass due to the new 202 for mute